src/lib.rs

```diff
@@ -384,6 +444,17 @@ pub mod basic;
 #[cfg(feature = "curl")]
 pub mod curl;
 
+///
+/// Device Code Flow OAuth2 implementation
+/// ([RFC 8628](https://tools.ietf.org/html/rfc8628)).
+///
+pub mod devicecode;
+pub use devicecode::StandardDeviceAuthorizationResponse;
```

Comment:
looks like the exports are a bit inconsistent right now. some of them (e.g., `StandardDeviceAuthorizationResponse`, `DeviceAuthorizationRequest`, `DeviceAccessTokenRequest`) are being exported at the top-level (oauth2::*). the others are being exported within the `oauth2::devicecode` module.

I don't have a strong opinion on which of the two is preferable here, but I think we should be consistent.

Reply:
Probably makes sense to just scope them to the `devicecode` module. Will fix up.